darc/darc.go

```diff
@@ -1012,6 +1025,25 @@ func NewIdentityX509EC(public []byte) Identity {
 	}
 }
 
+// NewIdentityECDSA creates a new ECDSA identity struct given a public key
+func NewIdentityECDSA(publicKey ecdsa.PublicKey) Identity {
+	return Identity{
+		ECDSA: &IdentityECDSA{
+			PublicKey: publicKey,
+		},
+	}
+}
+
+//TODO make calls to tsm  available
```

Comment:
Please resolve all TODOs.

From what I understand, the `Verify` method will not call to the TSM, no?